extension/src/webview/workspace.ts

```diff
@@ -27,14 +27,14 @@ export abstract class BaseWorkspaceWebviews<
     }
   }
 
-  public async showWebview(overrideRoot?: string) {
+  public async showWebview(overrideRoot: string, viewColumn?: ViewColumn) {
```

Comment:
We couldn't have both `overrideRoot` and `viewColumn` be optional so I decided to make the `overrideRoot` argument be an empty string if undefined.